app/views/course/achievement/course_users/_course_users_form.html.slim

```diff
@@ -2,37 +2,16 @@
   = f.error_notification
   = f.error :achievement
 
-  - collection = course.course_users.without_phantom_users.students.order_alphabetically
-  table.table.table-hover
-    thead
-      th = t('.name')
-      th
-        input.toggle-all id="toggle-all-normal-achievements" type='checkbox' style='display: none'
-        '
-        label for='toggle-all-achievements'
-          = t('.enabled')
-    tbody
-      = f.collection_check_boxes :course_user_ids, collection, :id, :name do |f|
-        tr
-          th = link_to_course_user(f.object)
-          td = f.check_box 'data-for-select-all' => "toggle-all-normal-achievements"
+  - own_students = course.course_users.without_phantom_users.students.where(creator_id: current_user.id)
```

Comment:
You should use `current_user.my_students` instead to get `own_students` instead of this query.